src/utils/useEdgeDrag.ts

```diff
@@ -3,6 +3,7 @@ import { EdgeSections } from '../symbols/Edge';
 import { NodeData, PortData } from '../types';
 import { DragEvent, NodeDragEvents, Position } from './useNodeDrag';
 import { Point2D } from 'kld-affine';
+import { NodeDragType } from 'symbols/Node';
```

Comment:
Need the .. pathing on it 

```suggestion
import { NodeDragType } from '../symbols/Node';
```